include/aedis/resp3/request.hpp

```diff
@@ -205,11 +207,18 @@ class request {
     *  
     *  @param cfg Configuration options.
     */
-   explicit request(config cfg = config{false, true, false, true})
-   : cfg_{cfg}
+   explicit request(boost::container::pmr::memory_resource * resource)
+         : payload_(resource), cfg_{false, true, false, true}
```

Comment:
AFAICS, we don't need this overload, the one below is enough. I would like to have `config{false, true, false, true}` twice in the code.